backend/molgenis-emx2-io/src/test/java/org/molgenis/emx2/io/TestRenameAndDelete.java

```diff
@@ -50,10 +50,10 @@ public void testRenameAndDrop() {
     store = new TableStoreForCsvInMemory();
     store.writeTable(
         "molgenis",
-        List.of("tableName", "columnName", "oldName"),
+        List.of("tableName", "columnName", "oldName", "key"),
```

Comment:
should have been a bug before, found it now because we started to add primary keys